docs/framework/network-programming/changes-to-ntlm-authentication-for-httpwebrequest-in-version-3-5-sp1.md

```diff
@@ -44,7 +44,7 @@ Security changes were made in .NET Framework version 3.5 SP1 and later that affe
   
  7. Quit Registry Editor, and then restart the IISAdmin service and run IISReset.  
   
- A less secure work around is to disable the loop back check, as described in [http://support.microsoft.com/kb/896861](https://go.microsoft.com/fwlink/?LinkID=179657). This disables the protection against reflection attacks. So it is better to constrain the set of alternate names to only those you expect the machine to actually use.  
+ A less secure work around is to disable the loop back check, as described in [https://support.microsoft.com/kb/896861](https://go.microsoft.com/fwlink/?LinkID=179657). This disables the protection against reflection attacks. So it is better to constrain the set of alternate names to only those you expect the machine to actually use.  
```

Comment:
please change this link to this format
`<https://support.microsoft.com/kb/896861>`